package.json

```diff
@@ -17,7 +17,7 @@
     "lint-fix": "eslint --fix --ext .ts .",
     "test": "node -r esbuild-register --test test/**/*.test.ts",
     "test:coverage": "nyc npm run test",
-    "release": "read -p 'GITHUB_TOKEN: ' GITHUB_TOKEN && export GITHUB_TOKEN=$GITHUB_TOKEN && release-it",
+    "release": "read -p 'GITHUB_TOKEN: ' GITHUB_TOKEN && export GITHUB_TOKEN=$GITHUB_TOKEN && release-it patch",
```

Comment:
```suggestion
    "release": "read -p 'GITHUB_TOKEN: ' GITHUB_TOKEN && export GITHUB_TOKEN=$GITHUB_TOKEN && release-it",
    "release:patch": "npm run release -- patch",
    "release:minor": "npm run release -- minor",
    "release:major": "npm run release -- major"
```

Reply:
Does your hint work also using it so `npm run release --pacht` ?

Reply:
Nope, should be: `npm run release -- patch`, note the space before and after `--`